js/ui/grid_core/ui.grid_core.virtual_scrolling.js

```diff
@@ -335,9 +335,11 @@ const VirtualScrollingRowsViewExtender = (function() {
 
                 scrollPosition = itemIndex * itemSize;
 
-                for(const index in itemSizes) {
-                    if(index <= itemIndex) {
-                        scrollPosition += itemSizes[index] - itemSize;
+                if(itemIndex > 0) {
+                    for(const index in itemSizes) {
+                        if(index <= itemIndex) {
+                            scrollPosition += itemSizes[index] - itemSize;
+                        }
```

Comment:
Shouldn't we break the loop if index is greater than itemIndex? May be it is better to interrupt it.
```js
for(const index in itemSizes) {
   if(index <= itemIndex) {
      scrollPosition += itemSizes[index] - itemSize;
   } else {
      break;
   }
}
```